integration/tests/epochs/suite.go

```diff
@@ -395,3 +387,18 @@ func (s *Suite) ExecuteReadApprovedNodesScript(ctx context.Context, env template
 
 	return v
 }
+
+func (s *Suite) WaitForEpochs(ctx context.Context, numOfEpochs int)  {
```

Comment:
Could we add a godoc to this function, in particular documenting how the function determines when we've reached the target epoch. Also, I think this function assumes it is invoked at epoch 0. Otherwise the condition on line 396 for the first iteration through the outer loop would never be true (eg. if this function is called in epoch >=1). Maybe we could change the function to `WaitForEpoch(epochCounter uint64)` and just remove the outer loop